modules/moa-create-dedicated-cluster-admins.adoc

```diff
@@ -5,10 +5,13 @@
 
 [id="moa-create-dedicated-cluster-admins.adoc"]
 = Creating dedicated-admins and cluster-admins
+As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. You can create additional `cluster-admin` or `dedicated-admin` users. `Dedicated-admin` users have fewer privileges.
 
 .Prerequisites
 
-You have added an Identity Provider (IDP) to your cluster.
+* You have added an Identity Provider (IDP) to your cluster.
+* You have the IDP user name for the user you are creating.
+* You are logged in to the cluster.
 
 .Procedure
```

Comment:
Since the `cluster-admin` role has more privileges, would it make sense to list the steps for creating `cluster-admin` roles before `dedicated-admin` roles? This isn't a pressing change request, just curious. :)